spring-boot-starter-camunda/src/main/java/io/camunda/zeebe/spring/client/configuration/ZeebeClientConfiguration.java

```diff
@@ -200,4 +200,8 @@ public boolean useDefaultRetryPolicy() {
     return properties.useDefaultRetryPolicy();
   }
 
+  @Override
+  public String toString() {
+    return "ZeebeClientConfiguration{" + "properties=" + properties + ", commonConfigurationProperties=" + commonConfigurationProperties + ", authentication=" + authentication + ", jsonMapper=" + jsonMapper + ", interceptors=" + interceptors + ", zeebeClientExecutorService=" + zeebeClientExecutorService + '}';
```

Comment:
Could this potentially leak credentials?

Reply:
True. Set all to hidden as far as I could identify them, see latest commit